frontend/pages/group/data/categories.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <div>
+    <!-- Create Dialog -->
+    <BaseDialog
+      v-model="state.createDialog"
+      :title="$t('data-pages.categories.new-category')"
+      :icon="$globals.icons.categories"
+      @submit="createCategory"
+    >
+      <v-card-text>
+        <v-form ref="domNewCategoryForm">
+          <v-text-field
+            v-model="createTarget.name"
+            autofocus
+            :label="$t('general.name')"
+            :rules="[validators.required]"
+          ></v-text-field>
+        </v-form>
+      </v-card-text>
+
+    </BaseDialog>
+
+    <!-- Edit Dialog -->
+    <BaseDialog
+      v-model="state.editDialog"
+      :icon="$globals.icons.categories"
+      :title="$t('data-pages.categories.edit-category')"
+      :submit-text="$tc('general.save')"
+      @submit="editSaveCategory"
+    >
+      <v-card-text v-if="editTarget">
+        <div class="mt-4">
+          <v-text-field v-model="editTarget.name" :label="$t('general.name')"> </v-text-field>
+        </div>
+      </v-card-text>
+    </BaseDialog>
+
+    <!-- Delete Dialog -->
+    <BaseDialog
+      v-model="state.deleteDialog"
+      :title="$tc('general.confirm')"
+      :icon="$globals.icons.alertCircle"
+      color="error"
+      @confirm="deleteCategory"
+    >
+      <v-card-text>
+        {{ $t("general.confirm-delete-generic") }}
+      </v-card-text>
+    </BaseDialog>
+
+    <!-- Recipe Data Table -->
+    <BaseCardSectionTitle :icon="$globals.icons.categories" section :title="$tc('data-pages.categories.category-data')"> </BaseCardSectionTitle>
+    <CrudTable
+      :table-config="tableConfig"
+      :headers.sync="tableHeaders"
+      :data="categories || []"
+      :bulk-actions="[]"
+      @delete-one="deleteEventHandler"
+      @edit-one="editEventHandler"
+    >
+      <template #button-row>
+        <BaseButton create @click="state.createDialog = true">
+          <template #icon> {{ $globals.icons.categories }} </template>
```

Comment:
I'm not sure if we need to change the icons for the create button, since the icon already appears right above it:
![image](https://github.com/mealie-recipes/mealie/assets/71845777/625e1712-efde-4eb7-b604-51fdf3a426ea)

Same for labels, tags, and tools

Reply:
![image](https://github.com/mealie-recipes/mealie/assets/24235032/ec190655-da4f-48d5-9d40-9c3bec45b49a)

resolved
Yeah i just wanted to keep it consistent with (most of) the existing pages. 

By the same logic i updated the combine button to use the externalLink icon (see above)